modules/dnn/src/ocl4dnn/include/ocl4dnn.hpp

```diff
@@ -450,6 +450,7 @@ struct OCL4DNNSoftmaxConfig
     MatShape in_shape;
     int axis;
     int channels;
+    bool logsoftmax;
```

Comment:
Please initialize members in ctor to make happy static code analyzers.

Reply:
Thanks for the review,  added member initialization in the constructor